src/styles/theme.scss

```diff
@@ -171,6 +175,27 @@
 	.vega-embed path {
 		transition: d 200ms;
 	}
+
+	// Overall visual rhythm.
+	.wp-block-group + figure {
+		margin-top: 3.5rem;
+	}
+	.wp-block-group:has(+ .wp-block-group.has-background) {
```

Comment:
:exploding_head: Wow, css comes at you fast. I had no idea this syntax was available yet.

Regardless, there's no styles in it, so it can probably be removed.

Reply:
Thanks for the catch